portability-spi-gateway/src/main/java/org/dataportabilityproject/gateway/action/Action.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dataportabilityproject.gateway.action;
+
+/**
+ * Handles an action in the API server.
+ */
+public interface Action<Request, Response> {
```

Comment:
is it intentional that this is defined in both portability-gateway and portability-spi-gateway?

Reply:
Done. Removed. Nope, we moved it to the SPI after the initial review, must have merged back in.